.circleci/config.yml

```diff
@@ -221,6 +221,7 @@ jobs:
       - store_artifacts:
           path: ./integration/logs
   feeds:
+    resource_class: large
```

Comment:
Is this needed?

Reply:
It failed on ENOMEM while I was testing. No harm in bumping the machine size.